src/pages/Login/Login.tsx

```diff
@@ -0,0 +1,199 @@
+import React, { useCallback, useEffect, useState } from "react";
+import "./Login.scss";
+import { Button, Form, Nav } from "react-bootstrap";
+import { Link, useNavigate } from "react-router-dom";
+import { useForm } from "react-hook-form";
+import { contactpattern, emailpattern } from "../../constants";
+import { useDispatch, useSelector } from "react-redux";
+import { login } from "../../redux/store/Authentication/actions";
+import { LoginStatus } from "../../models/LoginStatus";
+
+function Login() {
+  const [tabState, SetTabState] = useState("with-email");
+  const [userMainField, setuserMainField] = useState("Email address");
```

Comment:
please maintain camelCase `SetTabState` should be `setTabState` similarly should have been `setUserMainField`

Reply:
Also we do not need to maintain 2 different state variables what we have here can be achieved using either of these variables

Reply:
updated name with camelCase and removed the unnecessary state used (userMainField)